server/mails/fr/mail_invitation.jade

```diff
@@ -36,7 +36,7 @@ html(style="height:100%;")
             tr(style="padding-top: 10px; text-align: center; font-size: 0.85em; font-style: italic; color: #999;")
               td
                 p
-                  | Envoyé depuis le&nbsp;
+                  | Envoyé depuis le 
```

Comment:
It's already here :) https://github.com/cozy/cozy-calendar/blob/master/.editorconfig#L15